examples/demo/basic/image_plot.py

```diff
@@ -79,7 +79,7 @@ class Demo(HasTraits):
         Group(
             Item(
                 "plot",
-                editor=ComponentEditor(size=size, bgcolor=bg_color),
+                editor=ComponentEditor(size=size, bgcolor=bgcolor),
```

Comment:
FTR : `ComponentEditor` is the only place where I see someone set `bgcolor` on an enable `AbstractWindow` - and it sets `bgcolor`, not the deprecated `bg_color`.